docs/content/concepts/resources.mdx

```diff
@@ -86,14 +84,33 @@ defs = Definitions(
 )
 ```
 
-</TabItem>
-<TabItem name="Using ops and jobs">
+Assets specify resource dependencies by annotating the resource as a parameter to the asset function.
+
+To provide resource values to assets, attach them to the <PyObject object="Definitions" /> objects. These resources are automatically passed to the function at runtime.
+
+### With sensors
+
+[Sensors](/concepts/partitions-schedules-sensors/sensors) can use resources in the same way as ops and assets, which can be useful for querying external services for data.
+
+To specify resource dependencies on a sensor, annotate the resource type as a parameter to the sensor's function. For more information and examples, refer to the [Sensors documentation](/concepts/partitions-schedules-sensors/sensors#using-resources-in-sensors).
+
+### With schedules
+
+[Schedules](/concepts/partitions-schedules-sensors/schedules) can also use resources in case your schedule logic needs to interface with an external tool or to make your schedule logic more testable.
 
-### Using ops and jobs
+To specify resource dependencies on a schedule, annotate the resource type as a parameter to the schedule's function. For more information and examples, refer to the [Schedules documentation](/concepts/partitions-schedules-sensors/schedules#using-resources-in-schedules).
+
+### With ops and jobs
+
+<Note>
```

Comment:
This callout feels a bit weird to me - thoughts?

Reply:
yeah i agree - struggling to put my finger on why though...

Reply:
I think it's because we're not explaining *why* they're considered advanced. Perhaps that's the wrong angle to take? Maybe we don't need to have this at all, since assets come first?

Reply:
I took it out for now - it's just too weird.